packages/eslint-plugin/src/rules/prefer-function-type.ts

```diff
@@ -105,13 +158,27 @@ export default util.createRule({
     function checkMember(
       member: TSESTree.TypeElement,
       node: TSESTree.Node,
+      tsThisTypes?: TSESTree.TSThisType[],
```

Comment:
nit just union in null here

```suggestion
      tsThisTypes?: TSESTree.TSThisType[] | null,
```